packages/web/src/scss/tools/_themes.scss

```diff
@@ -62,13 +62,22 @@
 
     // Toggle
     --#{globals.$prefix}toggle-input-mark-background-image-url: #{svg.escape(
-            url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="52" height="44" viewBox="0 0 52 44" fill="none"><g xmlns="http://www.w3.org/2000/svg" filter="url(#a)"><circle xmlns="http://www.w3.org/2000/svg" cx="26" cy="18" r="10" fill="#{map.get($themed-tokens, colors, action, toggle, content-basic)}" xmlns="http://www.w3.org/2000/svg"><rect x="0" width="8" height="1" rx="0.5" /><rect x="0" y="5" width="8" height="1" rx="0.5" /><rect x="0" y="10" width="8" height="1" rx="0.5" /><rect x="0" y="15" width="8" height="1" rx="0.5" /></svg>')
+            url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20" fill="none"><rect x="0" y="0" width="20" height="20" fill="#{map.get($themed-tokens, colors, action, toggle, selected-content)}"/></svg>')
         )};
     --#{globals.$prefix}toggle-input-mark-disabled-background-image-url: #{svg.escape(
-            url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="52" height="44" viewBox="0 0 52 44" fill="none"><circle xmlns="http://www.w3.org/2000/svg" cx="26" cy="18" r="10" fill="#{map.get($themed-tokens, colors, disabled, foreground)}"/></svg>')
+            url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20" fill="none"><rect x="0" y="0" width="20" height="20" fill="#{map.get($themed-tokens, colors, disabled, foreground)}"/></svg>')
```

Comment:
Do we need images to draw a rectangle? Isn't a background color just fine?

Reply:
It would work, but this way it behaves the same in both variations - with indicators and without. 

But we can do it your way 🤷